<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawFormatDialog</class>
 <widget class="QWidget" name="RawFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupFormat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Image format</string>
   </property>
   <widget class="QComboBox" name="comboColor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboType">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color space</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data type</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Image size</string>
   </property>
   <widget class="QLineEdit" name="lineEditWidth">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHeight">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
